hugo/layouts/partials/site/header.html

```diff
@@ -62,11 +62,36 @@
                 {{- end -}}
             </div>
 
-            <div class="header__search">
-                <a href="{{ "/search" | relLangURL }}" class="button button--icon button--white-simple" rel="search">
+            <div class="header__search" data-menu>
+                <a href="{{ "/search" | relLangURL }}" class="button button--icon button--white-simple" rel="search" data-menu-dropdown="header-dropdown-search">
                     {{- partial "icon.html" (dict "icon" "search" "class" "button__icon") -}}
                     <span>{{ T "ui_search" }}</span>
                 </a>
+
+                <div class="header__dropdown header__dropdown--search" data-menu-dropdown-target="header-dropdown-search">
+                    <p class="header__search-title">{{ T "ui_search_question" }}</p>
+
+                    <form action="/search" method="get">
```

Comment:
see comment above: consider moving this to a partial because we now have duplicate code (with just a small change in classname I think) in 3 places